framework/test/jest/unit/specs/modules/chain/synchronizer/fast_chain_switching_mechanism/fast_chain_switching_mechanism.spec.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	FastChainSwitchingMechanism,
+} = require('../../../../../../../../src/modules/chain/synchronizer/fast_chain_switching_mechanism');
+const {
+	Block: blockFixture,
+} = require('../../../../../../../mocha/fixtures/blocks');
+
+describe('fast_chain_switching_mechanism', () => {
+	afterEach(async () => {
+		jest.clearAllMocks();
```

Comment:
Jest already clears all mocks after each run so this line can be removed

Reply:
It didn't cleared so why I have to add it here.

Reply:
@yatki do you know why is this happening?

Reply:
@limiaspasdaniel @nazarhussain Because mock file assignments are not in beforeEach blocks. I didn't enable `clearMocks` rule intentionally in jest.config.js to be aware and avoid these kind of assignments. I explained it more above on shu's comment.

Reply:
As I stated above, if you could move jest.fn inside beforeEach blocks, you wouldn't need this call.